cnf-certification-test/networking/suite.go

```diff
@@ -243,3 +238,26 @@ func testOCPReservedPortsUsage(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+func testDualStackServices(env *provider.TestEnvironment) {
+	var nonCompliantServices []*corev1.Service
+	var errorServices []*corev1.Service
+	ginkgo.By("Testing services (should be either single stack ipv6 or dual-stack)")
+	for _, s := range env.Services {
+		result, err := services.IsDualStack(s)
+		if err != nil {
+			tnf.ClaimFilePrintf("%s", err)
+			errorServices = append(errorServices, s)
+		}
+		if !result {
+			nonCompliantServices = append(nonCompliantServices, s)
+		}
+	}
+	if len(nonCompliantServices) > 0 {
+		tnf.ClaimFilePrintf("Non compliant services:\n %s", services.ToStringSlice(nonCompliantServices))
+		ginkgo.Fail(fmt.Sprintf("Found %d non compliant services (either non single stack ipv6 or non dual-stack)", len(nonCompliantServices)))
+	}
+	if len(errorServices) > 0 {
+		tnf.ClaimFilePrintf("Non compliant services:\n %s", services.ToStringSlice(errorServices))
```

Comment:
Shouldn't this message be related to the services with error?

Reply:
fixed